docs/release_notes.rst

```diff
@@ -4,6 +4,14 @@ Release Notes
 **pydocstyle** version numbers follow the
 `Semantic Versioning <http://semver.org/>`_ specification.
 
+2.2.0 - not yet released
```

Comment:
Just put "Current Development Version" in the title instead of specifying the actual version.

Reply:
Done.